addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -452,7 +452,7 @@ return AbstractRenderer.extend({
                 var view = info.view;
                 self._addEventAttributes(element, event);
                 if (view.type === 'dayGridYear') {
-                    const color = this.getColor(event.extendedProps.color_index);
+                    const color = self.getColor(event.extendedProps.color_index);
```

Comment:
should not this be fixed in an earlier version ? (I know the fix is straightforward, but obvisouly the use case is not tested....)

Reply:
This fix seems to be fixed by this [commit](https://github.com/odoo/enterprise/commit/6e547541eb0d78b21b8942d2418d336392b5ae9a). I will remove my commit about this.